plugins/modules/proxmox_kvm.py

```diff
@@ -482,6 +482,22 @@
       - Timeout for operations.
     type: int
     default: 30
+  tpmstate0:
+    description:
+      - A hash/dictionary of options for the Trusted Platform Module disk.
+      - A TPM state disk is required for Windows 11 installations.
+    suboptions:
+      storage:
+        description:
+          - C(storage) is the storage identifier where to create the disk.
```

Comment:
```suggestion
          - O(tmpstate0.storage) is the storage identifier where to create the disk.
```

Reply:
Can you explain this change? I can't find any other instances of `- O(` in the same file. I'm basing my contribution off of what is already there for `efidisk0`, shouldn't this be largely the same with regard to documentation?

Reply:
That's because semantic markup (https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#semantic-markup-within-module-documentation) is new and will only be allowed from community.general 7.1.0 on.

In any case, `C()` is wrong here, even with classical markup it should be `I()` since this is an option name, and not a value.

Reply:
Tests failing with your suggestion. Please advise.